viral/grosmark_analysis.py

```
@@ -240,37 +309,97 @@ def offline_correlations(
+        plt.xlabel("Distance between peaks")
+        plt.ylabel("Average pearson correlation")
+        plt.title(f"Fit pearson corrleation r = {r:.2f}, p = {p:.2f}")
+        # plt.savefig("plots/correlations_peak_distance.png", dpi=300)
+    else:
+        offline_spks_pre, offline_spks_post = get_frozen_wheel_flu(
+            flu=spks, wheel_freeze=wheel_freeze
+        )
+        pre_corrs_real = get_offline_correlation_matrix(
+            offline=offline_spks_pre, wheel_freeze=True, do_shuffle=False, plot=True
+        )
+        pre_corrs_shuffled = get_offline_correlation_matrix(
```

Ruff (0.12.2)

Local variable `pre_corrs_shuffled` is assigned to but never used

Remove assignment to unused variable `pre_corrs_shuffled`

(F841)